magento/pwa-studio
# run dev server on docker env

## Description
docker dev-server is not working.

## Related Issue





Closes #1504 

## Verification Steps

Clone fresh repo and checkout `depatil/fix-docker-issue`
1. `bash docker/run-docker` should be successful and venia should be accessible at https://pwa-docker.localhost
2. `yarn watch:venia`  should be successful and venia should be accessible at https://magento-venia-concept-frnbw.local.pwadev:8745
3.  `yarn watch:all`  should be successful and venia should be accessible at https://magento-venia-concept-frnbw.local.pwadev:8745
4. `yarn stage:venia` should be successful and venia should be accessible at https://magento-venia-concept-frnbw.local.pwadev:9745/
